di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/handlers/BasicAuthHandler.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.di.ipv.stub.core.handlers;
+
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+import spark.Spark;
+import uk.gov.di.ipv.stub.core.config.CoreStubConfig;
+
+import java.util.Base64;
+
+public class BasicAuthHandler {
+    private static int NUMBER_OF_AUTHENTICATION_FIELDS = 2;
+    private static final String AUTHORIZATION_HEADER = "Authorization";
+    private static final String AUTHORIZATION_TYPE = "Basic";
+
+    public BasicAuthHandler() {
+        CoreStubConfig.getUserAuth();
```

Comment:
I think, we should read to config value and then set to CORE_STUB_BASIC_AUTH value in initRoutes() with second if. So, if CORE_STUB_BASIC_AUTH's value is null. We will throw exception and add some logs about that.